src/main/java/ShortSet.java

```diff
@@ -2,12 +2,21 @@
 
 public class ShortSet {
   public static void main (String[] args) {
+    int a;
+    int b;
     Set<Short> s = new HashSet<Short>();
     for (short i = 0; i < 100; i++) {
+      int c;
```

Comment:
Please don't use single character variable names...
```suggestion
      int count;
```